include/simdjson/generic/ondemand/document.h

```diff
@@ -318,6 +318,9 @@ class document {
    * Returns debugging information.
    */
   inline std::string to_debug_string() noexcept;
+
+  simdjson_really_inline simdjson_result<value> at_pointer(std::string_view json_pointer) noexcept;
```

Comment:
If this is a new public method, then it should be documented (with a comment above).